shared/js/ui/templates/hamburger-menu.es6.js

```diff
@@ -14,7 +14,7 @@ module.exports = function () {
             <li>
                 <a href="javascript:void(0)" class="menu-title js-hamburger-menu-options-link">
                     Settings
-                    <span>Manage whitelist and other options</span>
+                    <span>Manage protection and other options</span>
```

Comment:
@jdorweiler should have caught this in the task, but "Protection" should be capitalized.